packages/core/mount-utils/core-mount-utils-browser-internal/index.ts

```diff
@@ -7,4 +7,3 @@
  */
 
 export { MountWrapper, mountReactNode } from './src/mount';
-export { KBN_LOAD_MARKS } from './src/consts';
```

Comment:
This was a duplicate const and only used within `@kbn/core-root-browser-internal`, so I removed it from here.

Reply:
Thanks! The duplicate was [temporary](https://github.com/elastic/kibana/pull/137365/files#r933008450) anyway.